lib/matplotlib/backends/backend_pgf.py

```diff
@@ -963,14 +963,12 @@ def get_renderer(self):
         return RendererPgf(self.figure, None, dummy=True)
 
 
-class FigureManagerPgf(FigureManagerBase):
-    pass
+FigureManagerPgf = FigureManagerBase
```

Comment:
I wouldn't do this because you don't create a `FigureManagerPgf` type with that, so that

~~~
>>> mgr = FigureManagerPgf()
>>> type(mgr)
<class 'FigureManagerBase'>
~~~

which is a bit confusing.

Reply:
this is consistent with FigureManagerPS, FigureManagerPDF and FigureManagerSVG, and was discussed in https://github.com/matplotlib/matplotlib/pull/9551#pullrequestreview-123540507.

Reply:
Still not convinced this is a good approach, but accepting as precedenced.